db/migrate/20240522010424_add_pet_status_to_application_pets.rb

```diff
@@ -0,0 +1,5 @@
+class AddPetStatusToApplicationPets < ActiveRecord::Migration[7.1]
+  def change
+    add_column :application_pets, :pet_status, :string
+  end
+end
```

Comment:
made this so you could change the status of a single application_pet without altering the pets on other apps